migration-engine/migration-engine-tests/tests/migrations/reset_tests.rs

```diff
@@ -58,7 +58,7 @@ fn reset_then_apply_with_migrations_directory_works(api: TestApi) {
         .assert_has_table("_prisma_migrations");
 }
 
-#[test_connector]
+#[test_connector(exclude(TiDB))]
```

Comment:
Why does this not work for TiDB?

Reply:
@Mini256  another one can you please respond ?

Reply:
I tried to solve this problem, but I found that when this testcase executes to the line `api.diagnose_migration_history(&dir).send_sync();`, in the absence of `_prisma_migrations` tables in both TiDB and MySQL MySQL will not throw an error, while TiDB will cause the test to terminate due to the following error thrown.

```
called `Result::unwrap()` on an `Err` value: ConnectorErrorImpl { user_facing_error: None, message: Some("Schema change caused error: [schema:1146]Table 'reset_then_diagnostics_with_migrations_directory_works._prisma_migrations' doesn't exist"), source: None, context: SpanTrace [] }
Schema change caused error: [schema:1146]Table 'reset_then_diagnostics_with_migrations_directory_works._prisma_migrations' doesn't exist
``` 

@janpio, Do you have any idea about this problem?